snapcraft.yaml

```diff
@@ -582,7 +582,7 @@ parts:
   minio:
     source: https://github.com/minio/minio
     source-type: git
-    source-tag: RELEASE.2023-10-07T15-07-38Z
+    source-tag: RELEASE.2023-11-20T22-40-07Z
```

Comment:
You didn't feel like jumping on the next next major version https://github.com/minio/minio/releases/tag/RELEASE.2023-12-02T10-51-33Z, I'm fine with waiting a few days :)

What a weird and annoying release tag naming scheme, yuk :)

Reply:
Yeah, I think I'll wait a bit for the "Major Refactor Release" version :)